multidict/_multilib/hashtable.h

```diff
@@ -1877,7 +1877,7 @@ md_traverse(MultiDictObject *md, visitproc visit, void *arg)
 static inline int
 md_clear(MultiDictObject *md)
 {
-    if (md->used == 0) {
+    if ((md->keys == &empty_htkeys) || (md->state == NULL)) {
```

Comment:
```suggestion
    if (md->keys == &empty_htkeys) {
```
`md->state` is never assigned to NULL by the code